src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Payment.java

```diff
@@ -19,6 +20,10 @@
 
     private String documentControlNumber;
 
+    private String status;
+
+    private LocalDateTime lastmodified;
```

Comment:
should be `Instant` or is LocalDateTime enough ?

Reply:
Tests did work with LocalDateTime. If we are using Instant will change it.

Reply:
Ok I see we are using Instance will change it.

Reply:
Changed to Instance.